### PR TITLE
fix: env is valid for js_test, js_binary and js_run_binary

### PR DESCRIPTION
**Pull request by gregmagolan**
# Description

We over-filtered this list.
 
# Type of change

- [x] Bug fix (change which fixes an issue)

# Test plan

How has this been tested?

- [x] Covered by existing tests cases

### CODE REVIEW

**Comment by gregmagolan** [comment]
Will add a test case in a follow-up.

**Review by Aghassi — APPROVED** [review]
Validated internally, this works thank you!

**Review by kormide — APPROVED** [review]

File: jest/defs.bzl [hunk 1]
```diff
@@ -237,7 +237,7 @@ def jest_test(
             key: val
             for key, val in kwargs.items()
             # List from https://bazel.build/reference/be/common-definitions#common-attributes-tests
```
Comment (jbedard) [inline jest/defs.bzl:239]: `env` is in this list though?

Comment (jbedard) [reply jest/defs.bzl:239]: Maybe add a comment explaining why this list is different then the link?

Comment (gregmagolan) [reply jest/defs.bzl:239]: yes, this is a filter out list. env should be passed through. will add comment

**Pull request merged**